dashboard/app/controllers/pd/workshop_user_admin_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Pd::WorkshopUserAdminController < ApplicationController
+  authorize_resource class: :pd_workshop_user_admin
+
+  # display the Facilitator search form, Facilitators found, and Courses for first Facilitator found
+  # assign / remove Courses actions redirect back to this form to display the updated course list
+  def facilitator_courses_form
```

Comment:
nit: we have a convention of adding a comment with the route above controller actions. [Example](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/api/v1/pd/workshops_controller.rb#L7). Not necessary, but it's helpful